reconcile/utils/terrascript_aws_client.py

```diff
@@ -4788,6 +4788,9 @@ def populate_tf_resource_alb(self, spec, ocm_map=None):
             target_name = t["name"]
             t_openshift_service = t.get("openshift_service")
             t_ips = t.get("ips")
+            t_protocol = t.get('protocol')
+            t_protocol_version = t.get('protocol_version')
+
```

Comment:
```suggestion
            t_protocol = t.get("protocol")
            t_protocol_version = t.get("protocol_version")

```

Reply:
fixed in [cf70df1](https://github.com/app-sre/qontract-reconcile/pull/3539/commits/cf70df1adf68654ca0f8c33976ab4308569fc7c1)